### PR TITLE
Enable updateParticipant(), updateParticipants(), subscribeToTracksAu…

### PR DESCRIPTION
**Pull request by kompfner**
…tomatically(), and setSubscribeToTracksAutomatically() in React Native

### CODE REVIEW

File: src/module.js [hunk 1]
```diff
@@ -266,7 +266,9 @@ const PARTICIPANT_PROPS = {
       if (callObject._preloadCache.subscribeToTracksAutomatically) {
         return false;
       }
-      if ([true, false, 'avatar'].includes(subs)) {
+      let validPrimitiveValues = [true, false];
+      !isReactNative() && validPrimitiveValues.push('avatar');
+      if (validPrimitiveValues.includes(subs)) {
```
Comment (mattieruth) [inline src/module.js:271]: i love reading your code because i never would have thought to do the check this way.

**Review by mattieruth — APPROVED** [review]
looks straightforward.

File: src/module.js (hunk 1 above)
Comment (kompfner) [reply src/module.js:271]: i'll take that as a compliment, though i'm not sure it is haha

Comment (mattieruth) [reply src/module.js:271]: Ah! It is! I'm sorry it didn't come off that way.

**Pull request merged**